ostd/src/arch/loongarch/trap/irq/eiointc.rs

```diff
@@ -0,0 +1,155 @@
+    ///
+    /// # Arguments
+    ///
+    /// * `core_num` - Number of processor cores
+    ///
+    /// # Note
+    ///
+    /// * Now only support single cpu, and the core num of cpu must less than or equal to 4
+    pub fn init(core_num: usize) {
+        assert!(core_num <= Self::MAX_CORE_NUM);
+        // Enable extended I/O interrupt
+        let mut v = iocsr_read_d(OTHER_FUNCTION_SETTING_REG);
+        v |= 1 << 48;
+        iocsr_write_d(OTHER_FUNCTION_SETTING_REG, v);
+        // Set encoding method
+        let mut v = iocsr_read_d(OTHER_FUNCTION_SETTING_REG);
+        v |= 1 << 49;
+        iocsr_write_d(OTHER_FUNCTION_SETTING_REG, v);
+        // Set interrupt pin routing
+        // 0..31    -> INT0
+        // 32..63   -> INT1
+        // 64..95   -> INT2
+        // 96..127  -> INT3
+        // 128..159 -> INT4
+        // 160..191 -> INT5
+        // 192..223 -> INT6
+        // 224..255 -> INT7
+        for i in 0..Self::MAX_EXT_IOI_MAP_NUM {
+            iocsr_write_b(Self::EXT_IOI_MAP_BASE + i, i as _);
+        }
+        // Set the interrupt to bounce on the cores on node 0
+        for i in 0..Self::MAX_INTERRUPT_NUM {
+            iocsr_write_b(
+                Self::EXT_IOI_MAP_CORE_BASE + i as usize,
+                (1 << core_num) - 1,
+            );
+        }
+        // Set the node type0 to node 0
+        iocsr_write_h(Self::EXT_IOI_NODE_TYPE_BASE, 0x01);
+        // Enable extended I/O interrupt
+        let mut v = iocsr_read_d(OTHER_FUNCTION_SETTING_REG);
+        v |= 1 << 48;
+        iocsr_write_d(OTHER_FUNCTION_SETTING_REG, v);
+    }
+
+    /// Enable a specific IRQ.
+    pub fn enable(irq: u8) {
+        let reg_no = irq as usize / Self::U64_BITS;
+        let bit_offset = irq as usize % Self::U64_BITS;
+        // Enable the interrupt
+        let mut enable = iocsr_read_d(Self::EXT_IOI_EN_BASE + reg_no * Self::U64_BYTES);
+        enable |= 1u64 << bit_offset;
+        iocsr_write_d(Self::EXT_IOI_EN_BASE + reg_no * Self::U64_BYTES, enable);
+        // Enable the bounce
+        let mut bounce = iocsr_read_d(Self::EXT_IOI_BOUNCE_BASE + reg_no * Self::U64_BYTES);
+        bounce |= 1u64 << bit_offset;
+        iocsr_write_d(Self::EXT_IOI_BOUNCE_BASE + reg_no * Self::U64_BYTES, bounce);
+    }
+
+    /// Disable a specific IRQ.
+    pub fn disable(irq: u8) {
+        let reg_no = irq as usize / Self::U64_BITS;
+        let bit_offset = irq as usize % Self::U64_BITS;
+        // Disable the interrupt
+        let mut enable = iocsr_read_d(Self::EXT_IOI_EN_BASE + reg_no * Self::U64_BYTES);
+        enable &= !(1u64 << bit_offset);
+        iocsr_write_d(Self::EXT_IOI_EN_BASE + reg_no * Self::U64_BYTES, enable);
+        // Disable the bounce
+        let mut bounce = iocsr_read_d(Self::EXT_IOI_BOUNCE_BASE + reg_no * Self::U64_BYTES);
+        bounce &= !(1u64 << bit_offset);
+        iocsr_write_d(Self::EXT_IOI_BOUNCE_BASE + reg_no * Self::U64_BYTES, bounce);
+    }
+
+    /// Claim an interrupt.
```

Comment:
[The summary line should be written in third person singular present indicative form. Basically, this means write “Returns” instead of “Return”.](https://rust-lang.github.io/rfcs/0505-api-comment-conventions.html#formatting)

Could you please fix all the summary line in the doc comments to follow this convention?

(You can ignore this rule if you want your comments to be consistent with those of other architectures. However, I don't believe that applies to this specific code.)